reaper/reaper/src/main/kotlin/com/emergetools/reaper/Manifest.kt

```diff
@@ -16,7 +16,11 @@ internal fun getApiKey(metadata: Bundle): String {
 }
 
 internal fun getBaseUrl(metadata: Bundle): String {
-  return metadata.getString(MANIFEST_TAG_OVERRIDE_BASE_URL, ReaperConfig.EMERGE_BASE_URL)
+  val baseUrl = metadata.getString(MANIFEST_TAG_OVERRIDE_BASE_URL, "")
+  check(baseUrl != "") {
+    "com.emergetools.OVERRIDE_BASE_URL must be set to a URL"
```

Comment:
i guess its not really an `override` if it has to be set.
Maybe we can have the error message link the doc you are creating?